app/config/config.yml

```diff
@@ -202,6 +202,7 @@ doctrine:
 swiftmailer:
     transport: %mailer_transport%
     host:      %mailer_host%
+    port:      %mailer_port%
```

Comment:
We have to update `parameters.yml.dist` and `parameters.yml.vagrant` to reflect this new parameter.

https://travis-ci.org/redelivre/login-cidadao/builds/416593768#L744